src/openrct2/ride/TrackPaint.cpp

```diff
@@ -1050,6 +1050,8 @@ void track_paint_util_right_helix_up_small_quarter_tiles_paint(
     }
 }
 
+// Need to change two numbers in here for the height (+ 28) Don't know whichone is which. This is to fix the Junior/Classic Mini
+// Coaster's boundbox to match tunnels).
```

Comment:
This is quite vague. What do you mean exactly?

Reply:
I need some help here. You can probably see the pattern in my changes. In this case I think the Junior coaster's the boundbox of the following need to be raised by 6 (6 is the finetuned number, 28 was only necessary for the minetrain)
```
track_left_half_banked_helix_up_small
Case 3;
    Case 3
Case 7;
    Case 0

track_right_half_banked_helix_up_small
Case 3;
    Case 0
Case 7;
    Case 3

track_left_half_banked_helix_up_large
Case 6
    Case 3
Case 13;
    Case 0

track_right_half_banked_helix_up_large
 Case 6;
    Case 0
 Case 13;
    Case 3
```
I do not know which coordinates are linked to what part of the track and I don't want to accidentally cause any bugs by raising the boundbox of some sprites too high (making things like trains glitch out)

Reply:
I will take care of it.